friends/serializers/friend_serializer.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional
+from rest_framework import serializers
+
+from auth_api.export_types.validation_types.validation_result import ValidationResult
+from auth_api.services.helpers import validate_user_email
+from friends.models.friend import Friend
+
+
+class FriendSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Friend
+        fields = "__all__"
```

Comment:
Serializer should be FriendRequestSerializer right?
and the model should be FriendRequest I guess?